jboss-kie-smartrouter/tests/bats/jboss-kie-smartrouter.bats

```diff
@@ -51,3 +57,61 @@ teardown() {
     unset JAVA_INITIAL_MEM_RATIO
     unset JAVA_MAX_MEM_RATIO
 }
+
+@test "verify the env vars when LOG_LEVEL is set" {
+    export LOG_LEVEL="WARN"
+    PACKAGES_LOG_LEVEL_CONF=org.xyz=INFO,com.acme=DEBUG
```

Comment:
for what this env stands for?
see: https://github.com/jboss-container-images/jboss-kie-modules/pull/373/files